### PR TITLE
Kinga_Straszkiewicz_LaTeX

### PR DESCRIPTION
**Pull request by kingastraszkiewicz**

### CODE REVIEW

File: main.tex [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+\usepackage{tipa}
+\usepackage{phonrule}
+\addbibresource{my_biblio.bib}
+\date{April 2019}
+
+\begin{document}
+
+\maketitle\
+
+\section{Section1}
```
Comment (MarcinFortuna-UG) [inline main.tex]: Why no space between section and the numeral?

File: main.tex [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    \item syntax
+    \item phonotactics
+    \item wordbuilding
+    \item programming
+    \item semantics
+\end{enumerate}
+\newline
+\section{Section2}
+\begin{IPA}
+\textipa{/"maI "neIm Iz "kIngA. "aI "aem @ "stju:d@nt @t @ jUnI"v3:s@ti @v "gdansk. "maI "Int@r@sts "Inklu:d lIn"gwIstIks, I"spES@li f@"n6l@dZi aend "graem@r, b@t "6lseU "6rIdZIn @v "laengwIdZ "aend "proUgraemIN./}
```
Comment (MarcinFortuna-UG) [inline main.tex]: 1. Please eliminate periods, there is no punctuation in transcription!
2. a university? The university.
3. in'clude
4. linguistics - velar nasal
5. there is a symbol for ash in tipa
6. also - long vowel; the diphthong looks different too
7. language, and, programming - correct the ash; language has a velar nasal

File: main.tex [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+\end{IPA}
+\vspace{3mm}
+\newline
+\begin{IPA}
+\textipa{"mam na "imjE "kjinga. "jEs\|[tEm s\|[tu"\|[dEn\|[tk\~O u\|[ni"vEr\|[s1\|[te\|[tu g\|[da\textltailn"scegO. \|[dO "mOix \|[zai\|[n\|[tErE"\|[sOva\textltailn \addspace\|[za"li\t{t\k{s}}a C\~E j\~E\|[z1kO"\|[z\|[na\|[s\|[tfO, a "\|[zvwa\k{s}\t{t\k{s}}a fO\|[nO"lOgja i grama"\|t1ka, "jak "ruv\|[niE\k{s} pOxO"\t{\|[d\|[z}E\textltailn E jE\|[n"\|[z1ka Ora\|[s prOgramO"va\textltailn E.}
```
Comment (MarcinFortuna-UG) [inline main.tex]: 1. uniwersytetu - which syllable will be stressed?
2. kjinga - please make [j] a diacritic
3. również - no [ni], please transcribe the patal nasal
4. of course, eliminate periods ;)

**Review by MarcinFortuna-UG — CHANGES_REQUESTED** [review]
Dear Kinga,

first of all, please rename the files so that they contain your name. Apart from this, please correct some minor things listed inline.

**Comment by MarcinFortuna-UG** [comment]
Dear Kinga, within the same pull request, please *remove* your old files in another commit. If I merged at this point, 4 new files would be added to the master branch.

**Review by MarcinFortuna-UG — APPROVED** [review]
👍

**Comment by kingastraszkiewicz** [comment]
Thank you :)

**Pull request merged**